extensions/amp-analytics/0.1/storage-impl.js

```diff
@@ -301,14 +305,20 @@ export class LocalStorageBinding {
 
   /** @override */
   loadBlob(origin) {
-    return new Promise(resolve => {
+    return new Promise((resolve, reject) => {
+      if (!this.win.localStorage) {
+        reject('No localStorage object found');
```

Comment:
yeah these do get caught already its just the type of `error` i wanted to clarify. (from logging pov). I probably won't merge these anyways and filter them from our logs to a new metric